### DISCUSSION TITLE
custom fields

### DISCUSSION THREAD

**Discussion by EltarieI** [pwndoc/pwndoc#198] [Q&A] [answered]
Hi,

I was working on the word template and tried to include custom fields for vulnerabilities and couldnt find how to do so.

Is there any way to include them?

Greetings
Sin

**Comment by EltarieI** [comment, marked as answer]
Nvm found it

**Reply by UserZer00** [reply]
Hey Sinmania

I'm also struggling to add custom fields to the report template. Could you maybe give me some hints on how to do so? 

Best regards 
Dave
